prototypes/ScheduledMerges.hs

```diff
@@ -715,14 +741,14 @@ data EventDetail =
          runsAtLevel   :: Int
        }
      | MergeStartedEvent {
-         mergePolicy   :: MergePolicy,
+         mergePolicy   :: Maybe MergePolicy,
          mergeType     :: MergeType,
-         mergeDebt     :: MergeDebt,
+         mergeDebt     :: Int,
```

Comment:
Too inconvenient?

Reply:
`MergeDebt` also contains the accumulated unspent credits, which will always be 0 when a merging run is created. So it seemed unneeded. But yeah, it should be `Debt` at least (which currently is just a synonym for Int).